.github/workflows/ci-requirements.yml

```diff
@@ -0,0 +1,34 @@
+name: SMARTS CI
```

Comment:
rename this file: `ci-formatting-and-dependencies.yaml`

Reply:
+1